deploy/Makefile

```diff
@@ -35,10 +25,37 @@ ps:
 	@docker-compose ps
 
 down: #.env
-	@[[ -f private/cega.yml ]] && [[ -f private/lega.yml ]] && docker-compose down -v || echo "No recipe to bring containers down\nHave you bootstrapped? (ie make bootstrap)"
+	@[[ -f private/lega.yml ]] && docker-compose down -v || echo "No recipe to bring containers down\nHave you bootstrapped? (ie make bootstrap)"
 
 clean:
 	rm -rf .env private
-	-docker network rm cega &>/dev/null
 
 clean-all: clean clean-volumes
+
+preflight-check:
+	@echo "Ensure the system is up"
+	sleep 120
+
+# Note: "sleep 120" will be replaced with a script that checks for open port, mq settings, etc.
+# It might be a good idea to run it in the same network as the containers.
+# docker run --rm -it --network=lega_lega \
+#        -v ${PWD}/integration/ensure_system_up.sh:/ensure_system_up.sh \
+#        nbisweden/ega-base /ensure_system_up.sh
+
+# For the moment, we simply do sleep 60, because we need
+# - the inbox notification server up
+# - the rabbitmq shovel to CentralEGA (the federated queue can be late, it doesn't hurt)
+# - the key server (the java spring needs 97 seconds, 10 are enough for the other)
+
+
+# Decrypt the encoded data from the repository
+# That is, some users private ssh keys and the database of fake CentralEGA Users
+prepare: ../tests/_common/dummy.sec ../tests/_common/john.sec ../tests/_common/jane.sec ../tests/_common/users.json
```

Comment:
maybe a different solution to prepare with `openssl` as I get:
```
╰─$ make prepare                                                                                                                                                                                               2 ↵
openssl enc -aes-256-cbc -d -in ../tests/_common/john.sec.enc -out ../tests/_common/john.sec -pass pass:1PbDCkTX5y8BnOjw
*** WARNING : deprecated key derivation used.
Using -iter or -pbkdf2 would be better.
bad decrypt
139748720580416:error:06065064:digital envelope routines:EVP_DecryptFinal_ex:bad decrypt:crypto/evp/evp_enc.c:537:
make: *** [../tests/_common/john.sec] Error 1

```

Reply:
Yes. Oscar had the same problem.
As this is a convenience target, please suggest a way to provide the files to the rest of the team. I can create a zip file and send it to Slack, and remove those lines. 
What do you say?

Reply:
I say tests need to be replicated by anyone cloning the repo, and I admit I have no good solution at this point for encrypting those files, besides using maybe encrypting with crypt4gh and having the key attached to travis in base64 maybe ... what say you ?

Reply:
hehe. Well. Let's not involve crypt4gh here.
I think we can figure something with openssl.
I'll pass some flag for better key derivation function, etc...
Can we do it in another PR? (since it currently works on Travis, and I can provide the files to the team)

Reply:
> Can we do it in another PR?

We can, but I would personally prefer if things are nice and work once we merge.

Reply:
In theory that `make prepare` should work for everybody, @silverdaz please let us have a better solution in a following PR.